samples/attestations/verify_attestation_chains_test.py

```diff
@@ -0,0 +1,329 @@
+    b'\x96\xfen\xa9\xf7\xa2\x1eW\xd3\xbd\n\x16\x12\xea8\xaa\x85xJ\x13d\xe5\x85'
+    b'\xdd\xe6\xca\x82;qw\xbe\x8fa\xb7\xeb\x06L\xd2\\pb\x0b\xbf\x9bj\xcc\xb0'
+    b'\x92\xf2\x81v\x1c\xa0\xa3?{~\x8e\xc1O\x1a\xc9\x7f\x9cCH\x1d\xef\x85\xe1'
+    b'\xeb\xa5\x08\x01\x00\x00')
+
+
+def test_verify_certificate():
+    signing_cert = x509.load_pem_x509_certificate(TEST_OWNER_ROOT,
+                                                  backends.default_backend())
+    issued_cert = x509.load_pem_x509_certificate(TEST_OWNER_PARTITION_CHAIN,
+                                                 backends.default_backend())
+    assert verify_attestation_chains.verify_certificate(signing_cert,
+                                                        issued_cert)
+
+
+def test_verify_certificate_fail():
+    signing_cert = x509.load_pem_x509_certificate(TEST_OWNER_ROOT,
+                                                  backends.default_backend())
+    issued_cert = x509.load_pem_x509_certificate(TEST_MANUFACTURER_ROOT,
+                                                 backends.default_backend())
+    assert not verify_attestation_chains.verify_certificate(signing_cert,
+                                                            issued_cert)
+
+
+def get_test_manufacturer_root():
+    return x509.load_pem_x509_certificate(TEST_MANUFACTURER_ROOT,
+                                          backends.default_backend())
+
+
+def get_test_owner_root():
+    return x509.load_pem_x509_certificate(TEST_OWNER_ROOT,
+                                          backends.default_backend())
+
+
+def make_temporary_file(contents):
+    """Creates a NamedTemporaryFile with contents and returns its file name.
+
+    Args:
+        contents: The contents to write to the temporary file.
+
+    Returns:
+        The name of the temporary file.
+    """
+    temp_file = tempfile.NamedTemporaryFile(delete=False)
+    temp_file.write(contents)
+    temp_file.close()
+    return temp_file.name
+
+
+@pytest.fixture(scope='function')
+def test_data():
+    mfr_root = make_temporary_file(TEST_MANUFACTURER_ROOT)
+    mfr_chain = make_temporary_file(TEST_MANUFACTURER_CHAIN)
+    owner_root = make_temporary_file(TEST_OWNER_ROOT)
+    owner_card_chain = make_temporary_file(TEST_OWNER_CARD_CHAIN)
+    owner_partition_chain = make_temporary_file(TEST_OWNER_PARTITION_CHAIN)
+    cert_chains = make_temporary_file(b'\n'.join([
+        TEST_MANUFACTURER_CHAIN,
+        TEST_OWNER_CARD_CHAIN,
+        TEST_OWNER_PARTITION_CHAIN
+    ]))
+    attestation = make_temporary_file(TEST_ATTESTATION)
+
+    param = {
+        'mfr_root': mfr_root,
+        'mfr_chain': mfr_chain,
+        'owner_root': owner_root,
+        'owner_card_chain': owner_card_chain,
+        'owner_partition_chain': owner_partition_chain,
+        'cert_chains': cert_chains,
+        'attestation': attestation
+    }
+    yield param
+    os.unlink(mfr_root)
```

Comment:
Files created by `NamedTemporaryFile` will be removed in destructor. Probably we may omit `os.unlink` calls.

Reply:
Done.